### PR TITLE
Modified 'Compiling Apps with .NET Native' topic

### PR DESCRIPTION
**Pull request by rpetrusha**
# Modified 'Compiling Apps with .NET Native' topic

## Suggested Reviewers

@Cloud-Writer, @mairaw

### CODE REVIEW

**Comment by rpetrusha** [comment]
@mairaw, I've added the WIP label so we can make sure this is reviewed by PSS before we merge.

File: docs/framework/net-native/index.md [hunk 1]
```diff
@@ -26,27 +26,30 @@ manager: "wpickett"
...
+-   Your apps feature the performance of native code. Usually, performance will be superior to code that is first compiled to IL and then compiled to native code by the JIT compiler. 
   
 -   You can continue to program in C# or Visual Basic.  
   
 -   You can continue to take advantage of the resources provided by the .NET Framework, including its class library, automatic memory management and garbage collection, and exception handling.  
   
  For users of your apps, [!INCLUDE[net_native](../../../includes/net-native-md.md)] offers these advantages:  
   
--   Fast execution times  
+-   Faster execution times for the majority of apps and scenarios
```
Comment (mairaw) [inline docs/framework/net-native/index.md]: nit: usually list items with 3 or more words need to have punctuation

File: docs/framework/net-native/index.md [hunk 2]
```diff
@@ -26,27 +26,30 @@ manager: "wpickett"
...
+-   Faster startup times for the majority of apps and scenarios 
   
 -   Low deployment and update costs  
   
 -   Optimized app memory usage  
-  
- But [!INCLUDE[net_native](../../../includes/net-native-md.md)] involves more than a compilation to native code. It transforms the way that .NET Framework apps are built and executed. In particular:  
+
+> [!IMPORTANT]
+> For the vast majority of apps and scenarios, .NET Native offers significantly faster startup times and superior performance when compared to an app compiled to IL or to an NGEN image. However, your results may vary. To ensure that your app has benefited from the performance enhancements of .NET Native, you should compare its performance with that of the non-.NET Native version of your app.
```
Comment (mairaw) [inline docs/framework/net-native/index.md]: can we include any links to where people can find out more about how they can do this comparison?

File: docs/framework/net-native/index.md [hunk 3]
```diff
@@ -26,27 +26,30 @@ manager: "wpickett"
   
  Typically, apps that target the .NET Framework are compiled to intermediate language (IL). At run time, the just-in-time (JIT) compiler translates the IL to native code. In contrast, [!INCLUDE[net_native](../../../includes/net-native-md.md)] compiles Windows apps directly to native code. For developers, this means:
```
Comment (mairaw) [inline docs/framework/net-native/index.md]: line 25 mentions Visual Studio 2015. shouldn't we say 2015 and later versions?

**Review by mairaw — APPROVED** [review]
LGTM. Left some comments if you wanna take a look @rpetrusha.

**Comment by rpetrusha** [comment]
Thanks, @mairaw. I've addressed all your comments.

**Comment by Cloud-Writer** [comment]
Awesome Ron!

Thanks,
Meera

On Oct 12, 2017, at 9:39 AM, Ron Petrusha <notifications@github.com<mailto:notifications@github.com>> wrote:


Merged #3318<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdotnet%2Fdocs%2Fpull%2F3318&data=02%7C01%7Cmeerak%40microsoft.com%7C3461e81a331a429fa63108d5118fc387%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636434231518851497&sdata=NwFpHY59rQwiN8RCX%2FDvblvYiFuRx1fMeq5dIMCtgjw%3D&reserved=0>.

**Pull request merged**